pyproject.toml

```
@@ -4,3 +4,9 @@ requires = ["Cython>=0.27.1", "setuptools", "wheel"]
 [tool.black]
 line-length = 100
 target-version = ['py36', 'py37', 'py38', 'py39']
+
+[tool.cibuildwheel]
+manylinux-x86_64-image = "manylinux2010"
+manylinux-i686-image = "manylinux2010"
+skip = "pp*"
```

We might need a skip on `*musllinux*` here too. I think 2.2.2 starts building musl libc wheels by default. We had to skip them on aer because it failed. Although thinking about it musl builds will probably work fine too since we don't have any issues like aer's build did around `yum` usage (which doesn't work in alpine linux).

---

Actually on second thought we'll need this skip. Mostly because retworkx and our other upstream deps probably aren't publishing musllinux binaries which means we'll need to build those from source. In the case of retworkx this will fail because I expect what ever alpine image their using doesn't come pre-installed with a rust compiler.

```suggestion
skip = "pp* *musllinux*"
```